packages/RdfService/__tests__/rdfservice.test.ts

```diff
@@ -148,7 +149,7 @@ describe("RdfService", () => {
       await obj.deleteNode("testSubject")
```

Comment:
Naive obs:

I have no reason to believe any action needs to be taken. But just for the sake of alignment:

In terms of API design, IF `deleteNode('a')` is not used far more than `deleteNode('a', true)` there is a case here for 2 separate functions (or not having a default value for `ignoreInboundReferences`):

1. `deleteNode`
2. `deleteNodeAndInbound`